test/Problems/SOC/soc_problems.jl

```diff
@@ -0,0 +1,169 @@
+function soc1_test()
+    #=
+        max 0x + 1y + 1z
+    s.t.
+        x == 1
+        x >= ||(y,z)||
+    =#
+
+    model = TestModel{Float64}()
+
+    x,y,z = MOI.add_variables(model, 3)
+
+    MOI.set(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1.0,1.0], [y,z]), 0.0))
+    MOI.set(model, MOI.ObjectiveSense(), MOI.MAX_SENSE)
+
+    ceq = MOI.add_constraint(model, 
+            MOI.VectorAffineFunction(
+                [MOI.VectorAffineTerm(1, MOI.ScalarAffineTerm(1.0, x))], [-1.0]),  
+                 MOI.Zeros(1))
+
+    vov = MOI.VectorOfVariables([x,y,z])
+
+    csoc = MOI.add_constraint(model, vov, MOI.SecondOrderCone(3))
+
+    return model
+end
+
+function soc2_test()
+    #=
+        max 0x + 1y + 1z
+      s.t.
+        x == 1
+        x >= ||(y,z)||
+    =#
+
```

Comment:
sometimes you leave an empty line after `=#` and sometimes you don't

Reply:
New style guide: Never leave empty spaces between =# and the model declaration.